response_operations_ui/forms.py

```diff
@@ -249,3 +249,43 @@ def validate_password(form, field):
         if password.isalnum() or not any(char.isupper() for char in password) or not any(char.isdigit() for char in
                                                                                          password):
             raise ValidationError('Your password doesn\'t meet the requirements')
+
+
+class RequestAccountForm(FlaskForm):
+    '''The form used by request-new-account.html'''
+    email_address = StringField('Enter the ONS email address to create an account for',
+                                validators=[InputRequired('Enter an email address'),
+                                            Email(message='Invalid email address'),
+                                            Length(max=254,
+                                                   message='Your email must be less than 254 characters')])
+    password = PasswordField('Enter the admin password',
+                             validators=[InputRequired('Enter the admin password')])
+
+    @staticmethod
+    def validate_email_address(_, field):
+        email = field.data
+        _validate_email_address(email)
+        local_part, domain_part = email.rsplit('@', 1)
+        if domain_part not in ['ons.gov.uk', 'ext.ons.gov.uk', 'ons.fake']:
+            logger.info('Account requested for non-ONS email address')
+            raise ValidationError('Not a valid ONS email address')
+
+
+class CreateAccountForm(FlaskForm):
+    first_name = StringField('First name', validators=[DataRequired(message='First name is required')])
```

Comment:
Validators should have a max size as well as data required ?

Reply:
They don't need a max size but if used for a field previously they should be consistent